components/area_office_map.js

```diff
@@ -14,35 +13,17 @@ import {
 
 const officeAddress = css`
   font-size: 12px;
-  font-family: 'Merriweather', Georgia, serif;
+  font-family: "Merriweather", Georgia, serif;
 `;
 const officeInfo = css`
   font-color: black;
   background-color: white;
+  margin: 2px;
 `;
 const officeName = css`
   font-size: 14px;
   font-weight: 500;
-  font-family: 'Merriweather', Georgia, serif;
-`;
-const button = css`
-  background-color: #3e57e2 !important;
-  color: white !important;
-  text-align: right;
-  text-transform: none !important;
-  font-size: 12px !important;
-  padding: 6px !important;
-`;
-
-const rightArrowIcon = css`
-  -moz-transform: scaleX(-1);
-  -o-transform: scaleX(-1);
-  -webkit-transform: scaleX(-1);
-  transform: scaleX(-1);
-  float: left;
-  filter: FlipH;
-  -ms-filter: Flip;
-  padding-right: 10px;
+  font-family: "Merriweather", Georgia, serif;
```

Comment:
don't need the quotes if it's just one word (I just learned about font families today! :))